GitCommitsForNonGitUsers.md

```diff
@@ -0,0 +1,42 @@
+#Git Commits For Non-Git Users
+
+This is a guide to aid others not familiar with git to contribute to our TIL repo.
+
+## FIRST TIME SETUP
+* copy clone to clipboard  
+![](http://images.thoughtbot.com/TIL/copyClone_s.jpg)
+* Run the following `commands` in your shell: `git clone <PASTE CLIPBOARD>`
+
+## TO SUBMIT
+* `git pull origin master`  
+* `git checkout -b <BRANCHNAME>`  
+* add files/folders locally
+* `git add .` to add everything in cur dir  
+* `git commit`  
+* press `i` to enter insert mode, write title & message  
+* `ESC :wq` to write-quit  
+* `git push origin <BRANCHNAME>`  
+* Compare & Pull Request on website
+
+## TO CHANGE
+* `git status` to see changed files in red
+* `git add .`
+* `git status` to see updated files in green
+* `git commit` ( -m "MESSAGE" to skip Vim)
+* `git push origin <BRANCHNAME>`
+
+## TO SQUASH
+* `git fetch origin`
+* `git rebase -i origin/master`
+* squash to 1 in vim; `:wq`
+* reword commit messages
+
+## TO MERGE
+* `git log`
+* `git push -f origin <BRANCHNAME>`
```

Comment:
I think this should go in the "To squash" section. You don't need to force push if you haven't squashed